sql/index/pilosa/driver.go

```
@@ -84,14 +84,18 @@ func (d *Driver) LoadAll(db, table string) ([]sql.Index, error) {
 	)
 	filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
-			errors = append(errors, err.Error())
+			if !os.IsNotExist(err) {
+				errors = append(errors, err.Error())
+			}
 			return filepath.SkipDir
 		}
 
 		if info.IsDir() && path != root && info.Name() != "." && info.Name() != ".." {
 			cfg, err := index.ReadConfigFile(path)
 			if err != nil {
-				errors = append(errors, err.Error())
+				if !os.IsNotExist(err) {
```

this should throw an error because the path exists but no the config file.